clarifaipyspark/dataset.py

```diff
@@ -337,7 +337,7 @@ def export_annotations_to_dataframe(self, input_ids: list = None, input_type: st
     response = list(self.list_annotations(input_ids=input_ids, input_type=input_type))
     for an in response:
       temp = {}
-      temp['annotation'] = str(an.data)
+      temp['annotation'] = str(an.data.regions) if an.data.regions else {}
```

Comment:
Text data might not have regions in annotations... they could be having some other field name

Reply:
great. I have included annotation.data.concepts as well to include text annotations. Thanks !